experimental/dds/tree2/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -155,21 +125,15 @@ export const makeFieldEditGenerator = (
 							fieldKey,
 							state.random.integer(0, count),
 							state.random,
-							state.treeIndex,
 						);
 					case "delete":
-						return generateSequenceFieldDeleteOp(
-							fieldPath,
-							state.random,
-							count,
-							state.treeIndex,
-						);
+						return generateSequenceFieldDeleteOp(fieldPath, state.random, count);
```

Comment:
this is pre-existing, but I just noticed that the `opWeightRatio` calculation above isn't right. the denominator should be the sum of the two options.

Reply:
updated `opWeightRatio`

Reply:
i think this still applies. I can't comment on it since GH doesn't let you make suggestions for unchanged lines, but L115 should be:

```
const opWeightRatio = passedOpWeights.insert / (passedOpWeights.insert + passedOpWeights.delete);
```

Reply:
oh, I see you updated the logic below. dedupe those if you can, and if you can't update both :P

Reply:
updated opWeightRatio